### PR TITLE
Implement new validators (array, boolean, greater, greaterEqual, less, lessEqual)

### PR DESCRIPTION
**Pull request by durmus-aydogdu**
I implemented new validators,
- array
- boolean
- greater
- greaterEqual
-  less
- lessEqual

### CODE REVIEW

**Comment by Frizi** [comment]
Please squash your changes. Those merge commits makes this hardly readable

**Comment by Frizi** [comment]
I see that you are also missing tests (spec files)

**Comment by 38elements** [comment]
I saw this change roughly.
There is the cosmetic change which is unnecessary.
The cosmetic change should be reverted.

File: build/dev-server.js [hunk 1]
```diff
@@ -28,15 +29,15 @@ var devMiddleware = require('webpack-dev-middleware')(compiler, {
 
 var hotMiddleware = require('webpack-hot-middleware')(compiler)
 // force page reload when html-webpack-plugin template changes
-compiler.plugin('compilation', function (compilation) {
```
Comment (38elements) [inline build/dev-server.js]: This is one of the cosmetic change which is unnecessary.

**Comment by durmus-aydogdu** [comment]
I fixed all issues.

**Comment by kevintruby** [comment]
Would love to see this implemented soon! I've used version 0.6.2 mainly in my past couple projects and was able to validate a simple true/false for checkbox section via the _required()_ validator, but that's been broken by the update to 0.7.4 treating false as valid.

**Comment by maxie7** [comment]
Hello, could you tell me, I have "vuelidate.min.js", but i need this file without minification?! (i just want to add my own validator with regexp. How/Where can i get it?

**Comment by kaleb4eg** [comment]
@Frizi , @38elements I also need few of these validations, could you merge that PR? That PR also will fix existing issue which I have linked few mins ago.

**Comment by Frizi** [comment]
@kaleb4eg unfortunately I can no longer do that, as I already explained here https://github.com/monterail/vuelidate/issues/369#issuecomment-445225102

**Comment by kaleb4eg** [comment]
@Frizi ths for explanation. Could you help me to reach smb from Monterail? Who exactly I can send a message to understand whether they will maintain it or not?

**Comment by cby016** [comment]
+1 for these additional validations

**Comment by literakl** [comment]
+1 for boolean, I need to check a GDPR consent checkbox and `required `does not work

**Comment by simonmaass** [comment]
when will this be released? looking for boolean validation for GDPR

**Comment by rajiv-aglab** [comment]
checkbox **required** validation is not working with "vuelidate": "^0.7.5", ? $invalid value remains false always

**Comment by dobromir-hristov** [comment]
False is a valid value. Write a custom isTrue validator for that.

**Pull request closed without merging**